dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -259,8 +262,13 @@ const getMedia = ({
 	mainMedia?: MainMedia;
 	canPlayInline?: boolean;
 	isBetaContainer: boolean;
+	isInLoopingVideoTestControl: boolean;
 }) => {
-	if (mainMedia?.type === 'LoopVideo' && canPlayInline) {
+	if (
+		mainMedia?.type === 'LoopVideo' &&
+		!isInLoopingVideoTestControl &&
```

Comment:
Perhaps I'm getting confused here, but wouldn't we want to hide looping videos to users in the variant (the 5%), rather than the control (the 95%)?